verification/verify_test.go

```diff
@@ -70,6 +74,17 @@ func TestVerify(t *testing.T) {
 	assert.Nil(err)
 	assert.NotNil(res)
 
+	// check pixel count fails w/o external verifier
```

Comment:
This is a good check for a failure condition. Might be good to also add another check that pixel count succeeds without an external verifier, probably by adapting some of the checks in TestTranscodeSegment_VerifyPixels (which can then be removed as per this [comment](https://github.com/livepeer/go-livepeer/pull/1360/files#r376504083)).

Reply:
Added in 14054cd1bcb8d94db6856a8be744d649ff7654de , but I had to add some ugly checks to determine whether to return `nil, err` or `params, err` ...  I'd imagine this might need some adjusting

Reply:
Actually, something is definitely wrong here.  The test passes, but I populated Params.Renditions with bogus data.

Reply:
...ugly checks removed